core/frontend/src/internal/tile/map/ImageryProviders/ArcGISImageryProvider.ts

```diff
@@ -99,6 +99,7 @@ export abstract class ArcGISImageryProvider extends MapLayerImageryProvider {
     if (this._accessTokenRequired && this._accessClient) {
       this._lastAccessToken = await ArcGisUtilities.appendSecurityToken(urlObj, this._accessClient, {
         mapLayerUrl: new URL(this._settings.url),
+        portal: typeof this._settings.properties?.portal === "string" ? this._settings.properties.portal : undefined,
```

Comment:
why is there a type check here?
why isnt just assigning portal to settings.properties.portal ?